xml/install_quick.xml

```diff
@@ -101,7 +101,99 @@
        </para>
       </important>
      </listitem>
+     <listitem>
+      <para> Ports </para>
+      <para> In a &productname; cluster, internal TCP/IP ports are managed using
+       iptables controlled by Salt and so need not be manually configured.
+       However, for reference and for environments where there are existing
+       security policies, the following are the standard ports in use.</para>
+      <note>
+       <title>SSH</title>
+       <para>SSH is not strictly necessary on all nodes.</para>
+      </note>
+      <itemizedlist>
+       <listitem>
+        <para>Internal</para>
```

Comment:
I think we have too many levels of lists and indentions, which make the text hard to read. Some ideas:
* Replace one level (e.g. Internal/External) with list titles.
* Don't use lists for the individual ports, just for the hosts (admin node/masters and workers)

Reply:
Or how about a table? Hosts as columns, ports as lines.